BUILD.md

```diff
@@ -295,7 +295,7 @@ $ ./sbt namerd/dcos:docker
 $ docker run -p 2181:2181 -p 4180:4180 -v /path/to/repo:/myapp -w /myapp buoyantio/namerd:0.9.1-SNAPSHOT-dcos namerd/examples/zk.yaml
 ```
 
-[funsuite]: http://www.scalatest.org/getting_started_with_fun_suite
+[funsuite]: https://www.scalatest.org/getting_started_with_fun_suite
```

Comment:
The HTTPS link for this give me a warning saying that the link is not secure. I think we can leave this as HTTP for now.